plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -69,6 +69,13 @@ func init() {
 		},
 	})
 	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "bootstrap-signer"},
+		Rules: []rbac.PolicyRule{
+			// other permissions for this controller are restricted per namespace
+			eventsRule(),
```

Comment:
does this generate events outside the namespace? I expected all the permissions to be in the namespaced role